libs/blocks/global-navigation/utilities/menu/menu.js

```diff
@@ -264,7 +264,8 @@ const decorateMenu = (config) => logErrorFor(async () => {
   if (config.type === 'asyncDropdownTrigger') {
     const pathElement = config.item.querySelector('a');
     if (!(pathElement instanceof HTMLElement)) return;
-    const res = await fetch(`${pathElement.href}.plain.html`);
+    const path = pathElement.href.replace(/(\.html$|$)/, '.plain.html');
```

Comment:
We have a few other places where we fetch `plain.html` (footer, gnav, breadcrumbs)
Do we need to consider those too?

Reply:
Thanks for the explanation, so the difference is that urls from metadata are not affected. For now, this is good 👍